packages/manager-react-components/src/components/datagrid/visibility/visibility-management.component.tsx

```diff
@@ -17,7 +17,7 @@ import { useTranslation } from 'react-i18next';
 export type ColumnsVisibility = {
   id: string;
   isDisabled: boolean;
-  label: string;
+  label: string | JSX.Element;
```

Comment:
I anticipate that in the future we might have labels other than strings (a checkbox for example).

Reply:
I would recommend using `React.ReactNode` instead of `string | JSX.Element` then